alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/validator/ConfigurationFieldValidator.java

```diff
@@ -51,76 +50,82 @@ public static ConfigurationFieldValidator fromJobFieldModel(JobFieldModel jobFie
 
     public ConfigurationFieldValidator(Map<String, FieldValueModel> mapToValidate) {
         this.fieldMap = mapToValidate;
-        statuses = new HashSet<>();
     }
 
-    public Set<AlertFieldStatus> getValidationResults() {
-        return statuses;
-    }
```

Comment:
Nothing about the other methods tells me I will need to call this at some point. With a more descriptive class name it would put me on the right track, but ultimately it would still be lore-based.